backend/src/apiserver/server/test_util.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Kubeflow Authors
+// Copyright 2018-2022 The Kubeflow Authors
```

Comment:
This file indeed had some changes this year. I guess it's ok then to update the copyright year.

Reply:
If the year wasn't updated during previous change, it's fine. There's nothing needs to be fixed. Adding the current year is optional per https://opensource.google/documentation/reference/copyright#the_year, which I've referenced in the past. 

Having such a diff in PR would always incur the questions on whether it's needed and correct. Since we already spend quite some time on this trivial thing, you can keep this change this time. In the future, let's avoid such unnecessary changes.